noxfile.py

```diff
@@ -292,7 +292,7 @@ def local(session):
         "noxfile.py",
     )
 
-    install(session, "cryptography @ .")
```

Comment:
Did you mean to revert this?

Reply:
Yeah. Turns out this syntax doesn't work with pip, so we can't actually switch until uv is usable.